content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,473 @@
+# How to build a task manager with angular.
+
+### Introduction
+As developers, it is common practice to build simple applications. This will help solidify one's knowledge about a particular programming language or framework. 
+
+In this article, we will build a simple task manager application (AKA To-do app) using [Angular](https://angular.io/docs). Angular is a JavaScript framework.
+
+### Table of content
+-  Prerequisites
+-  Introduction
+-  Key takeaways
+-  What is a Task Manager
+-  Create a new angular project
+-  Create a component
+-  Add task
+-  Mark task as done
+-  Delete a task
+-  Save to local storage
+-  Summary
+
+
+### Prerequisites
+Before we get started, here are a few prerequisites for developing a task manager via Angular;
+
+- You should have [Node.js](https://nodejs.org/en/) installed on your computer.
+- You should be fairly familiar with [Angular](https://angular.io/docs) and JavaScript. Knowledge of [TypeScript](https://www.typescriptlang.org/) is useful but not requisite.
+
+### Key takeaways
+At the end of this tutorial, you should be able to:
+-  create and structure an angular project.
+-  create a component in Angular.
+-  build a simple application.
+-  save items to local storage
+
+
+### What is a Task Manager?
+A task manager writes, organizes, and rearranges tasks more efficiently. The task manager we will build will allow the user to add a task, mark the task as complete, and delete the task.
+
+
+### Create a new angular project
+Before creating a new angular project, one will need to install angular CLI. Run `npm install -g @angular/cli` on the terminal or command prompt to install the CLI.
+
+   ```npm install -g @angular/cli
+   ```
+
+After installing the CLI, you can now create a new angular project. Run the CLI command `ng new *project name*` to do this. We will use `todoApp` as the project name for this article.
+
+    ``` ng new todoApp 
+    ```
+ Accept the defaults by pressing the enter key.
+
+The Angular CLI will install the required Angular npm packages and other dependencies for the project.
+
+The Angular CLI has a server to serve and build applications locally.
+To serve your application to the browser, you will first navigate to the workspace folder. Run `cd todoApp` to do so. 
+Next, run the command `ng serve -o` and wait for some minutes; your application will launch in the browser.
+
```

Comment:
These 2 lines could be reduced to one: For example:

```To serve your application to the browser, cd into the project directory and run the following commands:```.
Avoid wordy sentences!